cmd/kubectl_token.go

```diff
@@ -109,6 +110,10 @@ func CredentialCommand() cli.Command {
 				Name:  "skip-verify",
 				Usage: "Skip verification of the CACerts presented by the Server",
 			},
+			cli.StringFlag{
+				Name:  "cache-dir",
+				Usage: "The absolute path to place or find a .cache directory containing the kubectl token (also via RANCHER_CACHE_DIR)",
```

Comment:
Add the `EnvVar` field here to specify the envvar and the cli will pull from there automatically
```suggestion
				Usage: "The absolute path for the kubectl-token .cache directory"
```